### PR TITLE
gitea

### PR DESCRIPTION
**Pull request by luto**

### CODE REVIEW

File: source/gitea.rst [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+  
+  .. image:: _static/images/gitea.png
+      :align: center
+
+#####
+Gitea
+#####
+
+Gitea_ is a painless self-hosted Git service written in Go and distributed under the MIT License, designed to provide
+similar core functionality as GitLab or GitHub, but at a much small resource footprint. The project started as a fork
```
Comment (SalocinHB) [inline source/gitea.rst]: Please change 'small' to 'smaller'

File: source/gitea.rst [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+of the then-popular Gogs_, but quickly behave more active and bigger than the original.
```
Comment (SalocinHB) [inline source/gitea.rst]: 'behave' -> 'became'

File: source/gitea.rst [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+============
+
+Like a lot of Go software, gitea is distributed as a single binary. Download it,
+verify the checksum specified in the respective ``.sha256`` file and finally
+make sure that the file can be executed.
+
+::
+
+  [gitea@stardust ~]$ mkdir ~/gitea
+  [gitea@stardust ~]$ wget -O gitea/gitea https://dl.gitea.io/gitea/1.3.2/gitea-1.3.2-linux-amd64
```
Comment (SalocinHB) [inline source/gitea.rst]: Please use a fake version number such as 42.23.11

File: source/gitea.rst [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+Point your browser to your gitea URL and finish the installation.
+
+Updates
+=======
+
+.. note:: Check the update feed_ regularly to stay informed about the newest version.
+
+Check Gitea's `releases <https://github.com/go-gitea/gitea/releases/latest>`_ for the latest version. If there is newer
+versions is available, repeat the "Installation" step followed by a restart using ``supervisorctl restart gitea``.
```
Comment (SalocinHB) [inline source/gitea.rst]: 'If there is newer versions is available' -> 'If a newer version is available'

Comment (luto) [reply source/gitea.rst]: much engrish, such plural, wow.. m)

File: source/gitea.rst [hunk 5]
```diff
@@ -144,8 +144,8 @@ Updates
 
 .. note:: Check the update feed_ regularly to stay informed about the newest version.
 
-Check Gitea's `releases <https://github.com/go-gitea/gitea/releases/latest>`_ for the latest version. If there is newer
-versions is available, repeat the "Installation" step followed by a restart using ``supervisorctl restart gitea``.
+Check Gitea's `releases <https://github.com/go-gitea/gitea/releases/latest>`_ for the latest version. If a newer
+Version is available, repeat the "Installation" step followed by ``supervisorctl restart gitea`` to restart gitea.
```
Comment (SalocinHB) [inline source/gitea.rst]: 'version' should be lower-case

**Pull request merged**